Sources/XcodesKit/SessionController.swift

```diff
@@ -0,0 +1,145 @@
+import PromiseKit
+import Foundation
+import AppleAPI
+
+public class SessionController {
```

Comment:
Can we rename this to `AppleSessionService` vs `SessionController`?

Reply:
Done 👍🏼